packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -383,7 +383,8 @@ export default class ScriptTransformer {
 
       if (
         e instanceof SyntaxError &&
-        e.message.includes('Unexpected token') &&
+        (e.message.includes('Unexpected token') ||
+          e.message.includes('Cannot use import')) &&
```

Comment:
I guess we can have `export` here as well now

Reply:
seems they haven't special cased that yet

![image](https://user-images.githubusercontent.com/1404810/66466055-54bf9300-ea82-11e9-9be4-42ec4b682c0a.png)